src/System.Runtime.WindowsRuntime.UI.Xaml/pkg/System.Runtime.WindowsRuntime.UI.Xaml.pkgproj

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+ 
+  <!-- The library actually works on desktop where WinRT is supported and it appears in Desktop.settings.targets
```

Comment:
This is no longer an issue since the support checks aren't driven by the *.settings.targets files.  Please remove this comment and delete `SkipSupportCheck`